Dockerfile

```
@@ -0,0 +1,94 @@
+
+# Set environment variables
+ENV PATH="/app/.venv/bin:$PATH"
+ENV HOME="/home/crawler"
+ENV PLAYWRIGHT_BROWSERS_PATH="/home/crawler/.cache/ms-playwright"
+
+# Install Playwright browsers as crawler user
+USER crawler
+RUN playwright install chromium
+
+# Switch back to root for remaining operations
+USER root
```

Hadolint (2.12.0)

[warning] Last USER should not be root

(DL3002)